### PR TITLE
Landing page components

### PR DESCRIPTION
**Pull request by alejandrok93**
# Description

Completed functionality and rough styling of main components for landing page: Featured Projects, PopularMakers, PopularReviewers

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status
- [x ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Tested by adding more data for each component
- [ ] Test A
- [ ] Test B

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My code has been reviewed by at least one peer
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Comment by alejandrok93** [comment]
Added Redux actions and reducers for the following components:
FeaturedProjects
PopularMakers
PopularReviewers

File: reactapp/src/reducers/landingPageReducer.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      id: 2,
+      name: "Tristen",
+      star_count: 4.2,
+      author: "john",
+      photo_url: "someURL.com"
+    }
+  ]
+};
+
+const landingPageReducer = (state = initialState, action) => {
```
Comment (tylarpierson) [inline reactapp/src/reducers/landingPageReducer.js]: Great job on implementing switch cases!

File: reactapp/public/index.html [hunk 2]
```diff
@@ -6,6 +6,7 @@
   <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <meta name="theme-color" content="#000000">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (michaelagard) [inline reactapp/public/index.html]: add comment what this stylesheet is for

File: reactapp/src/components/MakerTile/MakerTile.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.project {
```
Comment (michaelagard) [inline reactapp/src/components/MakerTile/MakerTile.css]: I was under the impression we should be using styled-components to style our components.

File: reactapp/public/index.html (hunk 2 above)
Comment (alejandrok93) [reply reactapp/public/index.html]: this is for the stars icon for each DIY project

File: reactapp/src/components/MakerTile/MakerTile.css (hunk 3 above)
Comment (alejandrok93) [reply reactapp/src/components/MakerTile/MakerTile.css]: oh ok, i think we can make changes to the design once we finalize that?

**Pull request merged**